commands/lxc2lxd.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package commands
+
+import (
+	"bytes"
+	"os"
+	"strings"
+
+	"github.com/juju/description"
+	"github.com/juju/errors"
+)
+
+//go:generate go run ../juju2/generate/filetoconst/filetoconst.go LXCMigrationScript ../../../lxc/lxd/scripts/lxc-to-lxd lxc2lxd_script.go 2017 commands
```

Comment:
No, it is grabbing it from the lxd github repo that we are using for lxd support.